### PR TITLE
Added check to remind users to save changes in their project

### PR DESCRIPTION
**Pull request by rwmtse**
@rcruz @bryanhiggins

### CODE REVIEW

File: lib/ui/views/accessList.js [hunk 1]
```diff
@@ -79,6 +79,7 @@ AccessListView = Backbone.View.extend({
 
         updateAccess(config, origin, subdomains);
         this.model.set("config", config);
+        window.views.projectInfo.trigger("dirty");
```
Comment (rcruz) [inline lib/ui/views/accessList.js]: I think we can just skip the event system and set projectInfo.dirty = true;

File: lib/ui/views/projectInfo.js [hunk 2]
```diff
@@ -180,11 +186,17 @@ ProjectInfoView = Backbone.View.extend({
...
+        this.dirty = true;
     },
     saveConfigToServer: function () {
+        var view = this;
         this.model.setConfigXml(function (result) {
+            view.dirty = false;
             alert(result.success ? "Successfully saved" : result.error);
         });
+    },
+    isDirty: function () {
```
Comment (rcruz) [inline lib/ui/views/projectInfo.js]: Don't think we need this. Can use view.dirty directly

**Comment by rwmtse** [comment]
@rcruz Skipped event

**Comment by rcruz** [comment]
r+

**Pull request closed without merging**